src/components/NcNoteCard/NcNoteCard.vue

```diff
@@ -143,6 +153,11 @@ export default {
 		--note-theme: var(--color-success);
 	}
 
+	&--info {
+		--note-background: var(--color-primary-light);
+		--note-theme: var(--color-primary);
```

Comment:
I think we need to introduce a separate variable `--color-info` here which uses `#006aa3` manually. Otherwise for Nextclouds themed in e.g. red, the info box looks like a warning box. 

This is the same issue for the infobox in Text for example, cc @max-nextcloud @mejo-

Reply:
I think `--color-primary-default` should work, but there is no `--color-primary-default-light` afaik.

Reply:
`--color-primary-default` can be overridden by custom themes, so I think adding a `--color-info` (and `--color-info-rgb` + `--color-info-hover` respectively) would be the best way to go.

Reply:
Oh, good catch @jancborchardt! I agree that we should introduce a `--color-info` variable! I opened https://github.com/nextcloud/text/issues/4161 to track this issue in Text.